docs/source/tutorials/outliers.ipynb

```diff
@@ -476,50 +491,140 @@
+    "\n",
+    "scaler = preprocessing.StandardScaler().fit(train_feature_embeddings)\n",
+    "train_feature_embeddings_scaled = scaler.transform(train_feature_embeddings)\n",
+    "test_feature_embeddings_scaled = scaler.transform(test_feature_embeddings)\n",
+    "\n",
+    "# Create model and use cross validation to generate held out pred-probs (may take a few minutes)\n",
+    "model = MLPClassifier(solver='adam', hidden_layer_sizes=(512, 64, 6), random_state=1)\n",
+    "num_crossval_folds = 3  # for efficiency; values like 5 or 10 will generally work better\n",
+    "train_pred_probs = cross_val_predict(\n",
+    "    model,\n",
+    "    train_feature_embeddings_scaled,\n",
+    "    train_labels,\n",
+    "    cv=num_crossval_folds,\n",
+    "    method=\"predict_proba\",\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "03e3f7b7",
+   "metadata": {},
+   "source": [
+    "Lets see how well our model fit by checking its accuracy on the train examples. We can also compute out-of-distribution predictions scores for the train dataset using cleanlab's `OutOfDistribution` class."
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "740a8774",
+   "id": "e110fc4b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "train_pred_labels = train_pred_probs.argmax(1)\n",
+    "accuracy = np.mean(train_pred_labels == train_labels)\n",
+    "print(f\"Model accuracy on held-out train_data {accuracy}\")\n",
+    "\n",
+    "OOD = OutOfDistribution()\n",
+    "train_ood_predictions_scores = OOD.fit_score(pred_probs=train_pred_probs, labels=train_labels)\n",
+    "\n",
+    "## Visualize top 15 most severe outliers in train data if desired\n",
+    "# top_idxs = (train_ood_predictions_scores).argsort()[:15]\n",
+    "# visualize_outliers(top_idxs, train_data)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "eb631419",
+   "metadata": {},
+   "source": [
+    "Out-of-distribution predictions scores can also be computed to estimate how likely additional test data are under the training data distribution. Here we first apply our trained classifier to obtain pred_probs for the test data and then pass these pred_probs into our already-fitted OutOfDistribution estimator. To do this, an `OutOfDistribution` estimator fitted on `train_pred_probs` can be used to score `test_pred_probs`.\n",
+    "\n",
+    "Visualizing the top 15 most severe outliers in test data we once again see many examples that are not animals. Some outliers were identified correctly! Since it was working with less information, using `pred_probs` for OOD detection performed worse than uing `feature_embeddings` as expected."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3f90d8a4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model = MLPClassifier(solver='adam', hidden_layer_sizes=(512, 64, 6), random_state=1)\n",
```

Comment:
instead of hardcoding the 6 here, it will be less confusing if at the top of the notebook, where you first defined `animal_classes`, you add line:
```
num_classes_indistribution = 6
```
and then use `num_classes_indistribution` in its place everywhere later.